packages/database/test/path.test.ts

```diff
@@ -117,5 +117,6 @@ describe('Path Tests', function() {
     expectGreater('/ab', '/a');
     expectGreater('/a/b', '/a');
     expectGreater('/a/b', '/a//');
+    expectGreater('/a/0971500000', '/a/00403311635');
```

Comment:
```suggestion
    expectGreater('/a/0971500000', '/a/00403311635');
    expectGreater('/a/0971500000', '/a/971500000');
```

Does this handle that case correctly (when both parse to the same Int, we fall back to comparing the length of the strings)?

Reply:
I think it should, but let's find out :)